tests/src/whisk/core/cli/test/ApiGwTests.scala

```diff
@@ -77,6 +72,114 @@ class ApiGwTests
         }
         finally {
             val deleteresult = wsk.api.delete(basepathOrApiName = testbasepath, expectedExitCode = DONTCARE_EXIT)
+            deleteresult.stdout should include("ok: deleted API")
```

Comment:
If you expect the delete to always succeed, a better alternative is to remove `expectedExitCode = DONTCARE_EXIT`.